src/legacy/core_plugins/management/public/np_ready/services/index_pattern_management/creation/config.ts

```diff
@@ -102,7 +102,7 @@ export class IndexPatternCreationConfig {
     return this.showSystemIndices;
   }
 
-  public getIndexTags() {
+  public getIndexTags(indexName: string) {
```

Comment:
Is there a reason not to delete this?

Reply:
Ties into rollup functionality which does return tags. Could probably be better expressed through class inheritance but that would involve a refactor of rollup code.